dapps-certification-persistence/src/IOHK/Certification/Persistence/Structure/Subscription.hs

```diff
@@ -0,0 +1,232 @@
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE LambdaCase                 #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE DuplicateRecordFields      #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE OverloadedLabels           #-}
+{-# LANGUAGE OverloadedLists            #-}
+{-# LANGUAGE OverloadedStrings          #-}
+
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+
+module IOHK.Certification.Persistence.Structure.Subscription where
+
+import Data.Maybe
+import Database.Selda
+import Database.Selda.SqlType as Selda
+import IOHK.Certification.Persistence.Structure.Profile
+import Data.Text hiding (drop)
+import Control.Exception ( throw)
+import Data.Aeson
+import Data.Swagger
+import Data.Char as Char
+import Control.Lens
+import Data.Proxy
+import Data.Int
+import qualified Data.Text as Text
+
+data FeatureType = L1Run | L2UploadReport
+                 deriving (Eq,Generic, Show, Bounded, Enum, Read)
+instance SqlType FeatureType
+
+instance ToJSON FeatureType where
+  toJSON = \case
+    L1Run -> "l1-run"
+    L2UploadReport -> "l2-upload-report"
```

Comment:
There might be two routes for Auditors, true L2 reports and just L0 reports. It's not going to change  a lot of things (just the type in the metadata and a field number) so maybe we can still have only 1 and ask the auditor to pick between the two when they're at that point

Reply:
if you think we should add `l0-upload-report` it can be easily added with small to almost no effort, at this point

Reply:
done